draft-js-mention-plugin/src/MentionSuggestions/index.js

```diff
@@ -58,7 +58,9 @@ export default class MentionSuggestions extends Component {
           focusedOptionIndex: size - 1,
         });
       }
-
+      if (!this.props.store.getAllSearches().has(this.activeOffsetKey)) {
```

Comment:
I'm bit out of the this code. Can you elaborate why this is solving the problem? At best add a comment so we know in the future when revisit it.